### PR TITLE
Ability to upload file and store it as data via landing page

### PR DESCRIPTION
**Pull request by jarmoza**
Addresses #1 . Files may now be uploaded and are stored in `fileText` in landing page (index.vue)'s data. Temporary text area is set up for viewing of file text content - though currently not pretty printed.

### CODE REVIEW

**Comment by jarmoza** [comment]
Of note: both Papaparse (file uploading and parsing) and file-saver (file downloading) packages are added via this commit.

File: pages/index.vue [hunk 1]
```diff
@@ -1,9 +1,79 @@
...
+				complete: results => {
+					this.sourceList = results.data[0];
+					console.log(results.data);
+					this.fileText = results.data;
+				},
+			});
+		}
+	},
+
+	// props: {
```
Comment (surchs) [inline pages/index.vue:72]: Are those comments from the tutorial? I.e. Can we remove them?

**Review by surchs — APPROVED** [review]
Thanks @jarmoza, this looks good!

File: pages/index.vue (hunk 1 above)
Comment (jarmoza) [reply pages/index.vue:72]: I'm remembering where and how to best use props. Just leftover commented out code. No props on the landing page thus far. Will cleanup in the next commit.

**Pull request merged**